docker/test/fasttest/Dockerfile

```diff
@@ -61,6 +61,7 @@ RUN apt-get update \
         software-properties-common \
         tzdata \
         unixodbc \
+        lsof \
```

Comment:
Then #19751 can be closed

Reply:
I merged your PR to keep your commit in history.